Src/LiquidProjections/ExceptionPolicy.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Threading.Tasks;
+
+namespace LiquidProjections
+{
+    public delegate Task ExceptionPolicy(ProjectionException exception, Task task);

```

Comment:
Please name the `task` variable to something like `projectionTask` and document the delegate. Also, I would just keep the delegate with the specific project. There's no need to have all of the projectors use the same delegate.